net/gatling/Makefile

```diff
@@ -0,0 +1,45 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gatling
+PKG_VERSION:=0.16
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/gatling-$(PKG_VERSION)
+PKG_SOURCE:=gatling-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.fefe.de/gatling/gatling-0.16.tar.xz
+PKG_HASH:=5f96438ee201d7f1f6c2e0849ff273b196bdc7493f29a719ce8ed08c8be6365b
+
+PKG_MAINTAINER:=Martin Hübner <martin.hubner@web.de>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/gatling
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Web Servers/Proxies
+	TITLE:=gatling - a high performance web server
+	URL:=https://www.fefe.de/gatling/
+	PKG_BUILD_DEPENDS:=openssl px5g-mbedtls libowfat
```

Comment:
This seems wrong to me.

Reply:
It looks a bit strange in the beginning, but if one omit the px5g-mbedtls, compilation will fail with this error:
```
pssl.c:9:10: fatal error: mbedtls/config.h: No such file or directory
    9 | #include <mbedtls/config.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make[3]: *** [GNUmakefile:139: pssl.o] Error 1
```
So I assume it was intended by the author, to have openssl and mbedtls mixed somehow.

Reply:
Interesting, but PKG_BUILD_DEPENDS should be elsewhere not in this section.

Reply:
Fixed in recent version. Hopefully I got it rigth.